evalml/tests/automl_tests/test_automl.py

```diff
@@ -957,27 +957,27 @@ def test_targets_data_types_classification(data_type, automl_type, target_type):
         pytest.skip("Skipping test where data type is numpy and target type is nullable dtype")
 
     if automl_type == ProblemTypes.BINARY:
-        X, y = load_breast_cancer()
-        if "bool" in target_type:
-            y = y.map({"malignant": False, "benign": True})
-
+        X, y = load_fraud(n_rows=200)
```

Comment:
Updating to use a subset of the fraud dataset instead, since that dataset has more interesting features (breast cancer dataset only has numeric)

Reply:
This test takes two minutes on this branch but 35 seconds on main. I think the reason is that the fraud dataset has and nlp column (`provider`). If we want to use fraud, maybe we should manually change that type since I don't think the `TextFeaturizer` is critical for verifying what the test is trying to check?

But on the other hand, why change the dataset at all? If point of the test is to check that automl can handle targets of different types, then the pipeline components (except the estimator) shouldn't matter? 

I think keeping the fraud dataset is fine but I wanted to give my two cents hehe

Reply:
Ah I think you're right; I had chosen to update to `load_fraud` on the basis that it was the fraud dataset which caught the error (because the dataset is more interesting) but given the premise of this test, will revert :D